test/unit/helpers/dom/AuthContainer.js

```diff
@@ -0,0 +1,15 @@
+define(['jquery', './Dom'], function ($, Dom) {
+
+  return Dom.extend({
+
+    authContainer: function () {
+      return this.el('auth-container');
+    },
+
+    hasClass: function (className) {
```

Comment:
nit: I think it would be nicer to have this test class contain the actual logic, i.e. instead of "hasClass", provide "canBeMinimized" which would return a boolean. This way the test itself doesn't need to know about the details of how this is set (i.e. some class being added).